src/components/cylc/Dot.vue

```diff
@@ -0,0 +1,88 @@
+<style>
+@import '~@/styles/status-indicator/resize.css';
+</style>
+
+<template>
+  <!--
+  The first group of classes below set the dot colour based on the task state.
+  The second group start, upon a state (hence colour) change for that task, and
+  stop, upon a user click onto that dot, the pulse animation.
+  -->
+  <status-indicator
+    :active="isActiveMappedState()"
+    :positive="isPositiveMappedState()"
+    :negative="isNegativeMappedState()"
+    :intermediary="isIntermediaryMappedState()"
+
+    v-on:click="dismissPulse()"
+    :pulse="isPulseOn"
+  >
+  </status-indicator>
+</template>
+
+<script>
+  // This script customises the default 'dot' icons and animations on them:
+  import 'status-indicator/styles.css'
```

Comment:
Ah, great spot, thank you! For context, there are actually two separate components ``status-indicator`` & ``vue-status-indicator`` (where the [creator specifies on the ``README.md``](https://github.com/coderdiaz/vue-status-indicator?ref=madewithvuejs.com) for the latter 'This is a fork of status-indicator with a few changes for use with Vue.js'), & [on my development branch by mistake I installed both instead of just the Vue version](https://github.com/cylc/cylc-ui/compare/master...sadielbartholomew:dot-view). And when I tidied up for this branch, I removed the non-Vue variation as I got build errors for it without it in my ``package-lock.json``, without realising, as it turns out from your observation, I was importing the wrong one. Then I was very confused as to why Travis threw an error in the same way on the ``status-indicator`` aspect, which I thought had gone completely from the code!

I'll get ``vue-`` prepended in my next commit.